src/main/java/nextstep/subway/path/application/ShortestPathFinder.java

```diff
@@ -0,0 +1,64 @@
+package nextstep.subway.path.application;
+
+import nextstep.subway.line.dto.LineSectionResponse;
+import nextstep.subway.line.dto.SectionResponse;
+import nextstep.subway.path.exception.SeperatedStationsException;
+import nextstep.subway.station.dto.StationResponse;
+import org.jgrapht.GraphPath;
+import org.jgrapht.alg.shortestpath.DijkstraShortestPath;
+import org.jgrapht.alg.shortestpath.KShortestPaths;
+import org.jgrapht.graph.DefaultWeightedEdge;
+import org.jgrapht.graph.WeightedMultigraph;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ShortestPathFinder {
+
+    WeightedMultigraph<String, DefaultWeightedEdge> graph;
```

Comment:
```suggestion
    private final WeightedMultigraph<Long, DefaultWeightedEdge> graph;
```
이렇게 사용하면 id를 String으로 변환하지 않아도 돼요 :)